### ISSUE TITLE
HumanComfortLib Error sqrt(x) for x<0

### ISSUE THREAD

**Issue by BorisMichaelsen** [OpenModelica/OpenModelica#8716]
### Description
With the Test1 of the attached example file I get the following error message:
```
Test1_05evt.c:110:53: error: use of undeclared identifier 'equationIndexes'
    throwStreamPrintWithEquationIndexes(threadData, equationIndexes, "Model error: Argument of sqrt(VEL.k) was %g should be >= 0", tmp7);
                                                    ^
Test1_05evt.c:204:55: error: use of undeclared identifier 'equationIndexes'
      throwStreamPrintWithEquationIndexes(threadData, equationIndexes, "Model error: Argument of sqrt(VEL.k) was %g should be >= 0", tmp16);
                                                      ^
Test1_05evt.c:262:55: error: use of undeclared identifier 'equationIndexes'
      throwStreamPrintWithEquationIndexes(threadData, equationIndexes, "Model error: Argument of sqrt(VEL.k) was %g should be >= 0", tmp25);
                                                      ^
3 errors generated.
mingw32-make: *** [<builtin>: Test1_05evt.o] Error 1
mingw32-make: *** Waiting for unfinished jobs....
mingw32-make: Leaving directory 'C:/Users/boris/AppData/Local/Temp/OPENMO~1/OMEdit/PMV~3.TES'
Compilation process failed. Exited with code 0x00000002.
```
### Steps to reproduce
I can solve the problem by writing sqrt(abs(v_air_in)) instead of sqrt(v_air_in) (see Test2 of the attached example). But this should not be necessary because v_air_in is constant 0.1.

### Expected behavior
The Test1 example should be executable and produce the same results as Test2.

### Additional files
[PMV.zip](https://github.com/OpenModelica/OpenModelica/files/8283336/PMV.zip)

### Version and OS

 - OpenModelica Version [run `omc --version` or check `Help->About OMEdit` from OMEdit]
Windows 10:
Connected to OpenModelica v1.18.0-dev-305-gdc48a0ce1a (64-bit)
Connected to OMSimulator v2.1.1.post73-gcab626e-mingw
WSL:
Connected to v1.19.0-dev-599-gca7559a46d
Connected to OMSimulator v2.1.1.post102-g532159b-linux-debug

 - Versions of used Modelica libraries if applicable
MSL 4.0.0
HumanComfortLib 2.13.0 build 1

 - OS: [e.g. Windows 10, 64 bit]
Windows 10, 64 bit
and
WSL

**Comment by AnHeuermann** [comment]
Some functions are missing an equation index in the generated code.  We use it for error messages to tell the user what equation causes a problem.

I'll see if I can find the correct location during code generation to add the index or add a dummy index if that is not possible.

**Comment by AnHeuermann** [comment]
Oh... it seems that this was not fixed in v1.19.0-dev-569-g4dbf5668cf (Windows) but is already running on v1.19.0-dev-689-g37eb8d967f (Ubuntu).

@BorisMichaelsen Can you try again with the latest nightly build?

**Comment by BorisMichaelsen** [comment]
Correct, in version v1.19.0-dev-689-g37eb8d967f the problem no longer occurs
